ext/hal/microchip/mec/mec1501/component/timer.h

```diff
@@ -216,24 +264,66 @@ typedef struct htmr_regs
+#define MCHP_CCT_CTRL_COMP0_CLR		(1ul << 25)	/* R/W1C */
+
+
+/* Capture Compare timer interrupt routing */
+#define MCHP_CCT_GIRQ		18u
+
+/* Bit position in GIRQ Source, Enable-Set/Clr, and Result registers */
+#define MCHP_CCT_TMR_GIRQ_POS	20u
+#define MCHP_CCT_CAP0_GIRQ_POS	21u
+#define MCHP_CCT_CAP1_GIRQ_POS	22u
+#define MCHP_CCT_CAP2_GIRQ_POS	23u
+#define MCHP_CCT_CAP3_GIRQ_POS	24u
+#define MCHP_CCT_CAP4_GIRQ_POS	25u
+#define MCHP_CCT_CAP5_GIRQ_POS	26u
+#define MCHP_CCT_CMP0_GIRQ_POS	27u
+#define MCHP_CCT_CMP1_GIRQ_POS	28u
+
+#define MCHP_CCT_TMR_GIRQ_VAL	(1ul << 20)
+#define MCHP_CCT_CAP0_GIRQ_VAL	(1ul << 21)
+#define MCHP_CCT_CAP1_GIRQ_VAL	(1ul << 22)
+#define MCHP_CCT_CAP2_GIRQ_VAL	(1ul << 23)
+#define MCHP_CCT_CAP3_GIRQ_VAL	(1ul << 24)
+#define MCHP_CCT_CAP4_GIRQ_VAL	(1ul << 25)
+#define MCHP_CCT_CAP5_GIRQ_VAL	(1ul << 26)
+#define MCHP_CCT_CMP0_GIRQ_VAL	(1ul << 27)
+#define MCHP_CCT_CMP1_GIRQ_VAL	(1ul << 28)
+#define MCHP_CCT_GIRQ_VAL_ALL	0x1FF00000ul
+
+/* Capture Compare timer GIRQ aggregated NVIC input */
+#define MCHP_CCT_NVIC_AGGR	10u
+
+/* Capture Compare timer direct NVIC inputs */
+#define MCHP_CCT_TMR_NVIC_DIRECT	146u
+#define MCHP_CCT_CAP0_NVIC_DIRECT	147u
+#define MCHP_CCT_CAP1_NVIC_DIRECT	148u
+#define MCHP_CCT_CAP2_NVIC_DIRECT	149u
+#define MCHP_CCT_CAP3_NVIC_DIRECT	150u
+#define MCHP_CCT_CAP4_NVIC_DIRECT	151u
+#define MCHP_CCT_CAP5_NVIC_DIRECT	152u
+#define MCHP_CCT_CMP0_NVIC_DIRECT	153u
+#define MCHP_CCT_CMP1_NVIC_DIRECT	154u
 
 /**
   * @brief Capture/Compare Timer (CCT)
   */
 typedef struct cct_regs
-{		/*!< (@ 0x40001000) CCT Structure	  */
-	__IOM uint32_t CTRL;	/*!< (@ 0x00000000) CCT Control		  */
+{
+	__IOM uint32_t CTRL;		/*!< (@ 0x00000000) CCT Control */
 	__IOM uint32_t CAP0_CTRL;	/*!< (@ 0x00000004) CCT Capture 0 Control */
 	__IOM uint32_t CAP1_CTRL;	/*!< (@ 0x00000008) CCT Capture 1 Control */
-	__IOM uint32_t FREE_RUN;	/*!< (@ 0x0000000C) CCT Free run counter  */
-	__IOM uint32_t CAP0;	/*!< (@ 0x00000010) CCT Capture 0	  */
-	__IOM uint32_t CAP1;	/*!< (@ 0x00000014) CCT Capture 1	  */
-	__IOM uint32_t CAP2;	/*!< (@ 0x00000018) CCT Capture 2	  */
-	__IOM uint32_t CAP3;	/*!< (@ 0x0000001C) CCT Capture 3	  */
-	__IOM uint32_t CAP4;	/*!< (@ 0x00000020) CCT Capture 4	  */
-	__IOM uint32_t CAP5;	/*!< (@ 0x00000024) CCT Capture 5	  */
-	__IOM uint32_t COMP0;	/*!< (@ 0x00000028) CCT Compare 0	  */
-	__IOM uint32_t COMP1;	/*!< (@ 0x0000002C) CCT Compare 1	  */
+	__IOM uint32_t FREE_RUN;	/*!< (@ 0x0000000C) CCT Free run counter */
+	__IOM uint32_t CAP0;		/*!< (@ 0x00000010) CCT Capture 0 */
+	__IOM uint32_t CAP1;		/*!< (@ 0x00000014) CCT Capture 1 */
+	__IOM uint32_t CAP2;		/*!< (@ 0x00000018) CCT Capture 2 */
+	__IOM uint32_t CAP3;		/*!< (@ 0x0000001C) CCT Capture 3 */
+	__IOM uint32_t CAP4;		/*!< (@ 0x00000020) CCT Capture 4 */
+	__IOM uint32_t CAP5;		/*!< (@ 0x00000024) CCT Capture 5 */
+	__IOM uint32_t COMP0;		/*!< (@ 0x00000028) CCT Compare 0 */
+	__IOM uint32_t COMP1;		/*!< (@ 0x0000002C) CCT Compare 1 */
```

Comment:
There's a ton of whitespace-only change going on here.  Would be cleaner to leave the space conventions unchanged.